frontend/src/components/developerEndpoint.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import axios from 'axios'
+import { Table, Container } from 'react-bootstrap';
+
+export interface Developer {
+    id: number;
+    first_name:string;
+    last_name:string;
+}
```

Comment:
Types and Interfaces should be kept in a separate file, promoting reusability and keeping files cleaner

This component is intended as transitory but good habits start from the beginning :)

Reply:
Moved it into DeveloperInterface file inside an Interface directory and exported it into the component.